src/SDK.ts

```diff
@@ -202,6 +202,9 @@ export default class SDK implements ISDK {
       } catch (error) {
         const elapsedTimeInMilliseconds = timer.milliSecondsElapsed;
         this.logWithLogger(LogLevel.ERROR, OCSDKTelemetryEvent.GETLWISTATUSFAILED, "Get LWI Details failed", requestId, undefined, elapsedTimeInMilliseconds, requestPath, method, error);
+        if (error.code === "ECONNABORTED") {
+          throw new Error(SDKError.ClientHTTPTimeoutErrorName + ":" + SDKError.ClientHTTPTimeoutErrorMessage);
```

Comment:
Can we use string interpolation?

const message = \`${SDKError.ClientHTTPTimeoutErrorName}: ${SDKError.ClientHTTPTimeoutErrorMessage}\`

Reply:
Should we have a util `throwClientHTTPTimeoutError`?

```
const throwClientHTTPTimeoutError = () => {
  const message = `${SDKError.ClientHTTPTimeoutErrorName}: ${SDKError.ClientHTTPTimeoutErrorMessage}`
  throw new Error(message);
}
```